exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -141,6 +161,47 @@ func withTestExporterConfig(fns ...func(*Config)) func(string) *Config {
 	}
 }
 
+// Opens Clickhouse client to `replicationEndpoint2` to check if table data was replicated.
+func checkReplicatedTableCount(t *testing.T, dbName string, tableNames []string, expectedCount int) {
+	// wait for replication
+	require.NotEmpty(t, tableNames)
+	time.Sleep(1 * time.Second)
+
+	config := withTestExporterConfig()(replicationEndpoint2)
+	client, err := newClickhouseClient(config)
```

Comment:
Will it depend on a local clickhouse server? Using a mock server like other UT tests. 
The real replicated table count is guaranteed by clickhouse server, We just verify if the query statement is correct.

Reply:
Will it depend on a local clickhouse server? - yes, it need to be start by running docker-compose.yml in example/replication_setup.

Reply:
I don't think this will pass in the current UT test pipeline, but this docker-compose.yml is very helpful for local dev.

Reply:
What would you propose to best handle tests? I just can't other way how to test it without creating clickhouse, zookeeper instances.

Reply:
As I said above, you can verify the query statement whether is your expectation when `ClusterName` or `TableEngine ` is configured. you can follow this:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/403626b4a4d402c89d19bf0def231db15cc17204/exporter/clickhouseexporter/exporter_logs_test.go#L80-L93

Reply:
The local test with real clickhouse server is useful but i do not known how to use it in the ci pipeline, may be we can add a build tag for only test in local? https://mickey.dev/posts/go-build-tags-testing/